infrastructure/terraform/kubernetes/outputs.tf

```diff
@@ -0,0 +1,3 @@
+output "cluster_name" {
+  value = module.eks.cluster_name
+}
```

Comment:
you mean all of them? https://github.com/terraform-aws-modules/terraform-aws-eks#outputs

Reply:
No, you are correct - not all of them. So that `cluster_name` is fine, perhaps we can also add the `kubeconfig` file, so that one can communicate with the created kubernetes cluster?